### PR TITLE
SOC-1546 | Thread listing confusing clickable area

### PR DESCRIPTION
**Pull request by rafalkalinski**
Fixes for the post action clickable area sizes, according to designer's guides:
https://wikia-inc.atlassian.net/browse/SOC-1546

### CODE REVIEW

File: front/styles/main/module/_discussion.scss [hunk 1]
```diff
@@ -24,8 +24,22 @@ $horizontal-padding: 15px;
 		font-size: $type-min;
 		font-weight: bold;
 
+		.upvote-area,
+		.replies-area {
+			color: #999;
```
Comment (bkoval) [inline front/styles/main/module/_discussion.scss]: Please, move this to a variable, or use an existing one.

File: front/styles/main/module/_discussion.scss [hunk 2]
```diff
@@ -24,8 +24,22 @@ $horizontal-padding: 15px;
...
+			height: 30px;
+			line-height: 30px;
+			padding: 0 2px;
+		}
+
 		.icon {
 			margin-right: 3px;
+
+			&.upvote {
+				margin-top: -4px;
```
Comment (bkoval) [inline front/styles/main/module/_discussion.scss]: Why?

Comment (bkoval) [inline front/styles/main/module/_discussion.scss]: At least add a comment :)

File: front/styles/main/module/_discussion.scss (hunk 1 above)
Comment (rafalkalinski) [inline front/styles/main/module/_discussion.scss]: All of the colors are done on the branch SOC-1266. I didn't want to do it here, because of conflicts.

**Comment by kvas-damian** [comment]
For one line posts ```discussion-content``` has only 20px height. It's extremely hard to hit it on mobile. Can we convert few pixels from margin into padding?

File: front/styles/main/module/_discussion.scss (hunk 2 above)
Comment (rafalkalinski) [inline front/styles/main/module/_discussion.scss]: comment added

**Comment by bkoval** [comment]
LGTM :+1:

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**